src/content/releases/release-v0.2.0.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: "Project Retrospectives, People Management, Notification Controls, and more"
+version: "0.2"
+date: 2024-10-04
+description: "TODO"
+published: true
+---
+
+Operately's latest release is packed with improvements designed to make your
+company’s operations smoother and more efficient. Here’s what’s new!
+
+## Project Retrospectives that drive results
```

Comment:
Weak and generic subtitle. It should emphasizes the benefits of doing project retros.